changelog.d/5325.bugfix

```diff
@@ -0,0 +1 @@
+Add account_validity's email_sent column to the list of boolean columns in synapse_port_db.
```

Comment:
please could you try to describe the improvement that the user will see, rather than the way that you are achieving that improvement?